nntrainer/compiler/tflite_opnode.cpp

```diff
@@ -42,11 +54,31 @@ void TfOpNode::setLayerNode(const LayerNode &layer) {
     }
     return v;
   };
+  /**
+   * Q1) Why convert from NCHW to NHWC?
+   * A1) the tflite uses NHWC format; nntrainer uses NCHW
+   *
+   * Q2) Why are only output tensors reshaped?
+   * A2) the tflite needs only one tensor between nodes. Therefore,
+   *basically, outputs are used for tflite tensors
+   **/
+  auto create_variables_with_NCHW_to_NHWC = [](auto tensor_getter,
+                                               unsigned size) {
+    Variables v;
+    v.reserve(size);
+    for (unsigned i = 0; i < size; ++i) {
+      Tensor *tensor = const_cast<Tensor *>(tensor_getter(i));
+      tensor->reshape(TensorDim{tensor->batch(), tensor->height(),
+                                tensor->width(), tensor->channel()});
+      v.push_back(tensor);
+    }
+    return v;
+  };
 
   inputs = create_variables(
     [&context](unsigned idx) { return &context.getInput(idx); },
     context.getNumInputs());
-  outputs = create_variables(
+  outputs = create_variables_with_NCHW_to_NHWC(
```

Comment:
I'm not sure why this is dealt with differently compared with input as in #L119?

Reply:
Ummm,, I would have to set same transform function, say OutputTransfromFn, to all the layers. But, the same thing can be easily done in `setLayerNode`. Let me know if I am wrong.